sys/linux/socket_alg_386.const

```diff
@@ -23,7 +23,7 @@ CRYPTO_ALG_TYPE_AKCIPHER = 13
 CRYPTO_ALG_TYPE_BLKCIPHER = 4
 CRYPTO_ALG_TYPE_CIPHER = 1
 CRYPTO_ALG_TYPE_COMPRESS = 2
-CRYPTO_ALG_TYPE_DIGEST = 14
+# CRYPTO_ALG_TYPE_DIGEST is not set
```

Comment:
What happened here?
It's still defined in:
https://elixir.bootlin.com/linux/v5.2.1/source/include/linux/crypto.h#L52
And all other CRYPTO_ALG_TYPE_* consts are still here. So there is something special about this one. What linux version did you use to generate constants? Was this const removed/renamed or something? Do we need to include something else to get this name?